modules/combined/test/tests/optimization/invOpt_mechanics/forward_and_adjoint.i

```diff
@@ -0,0 +1,219 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 2
+  xmin = 0.0
+  xmax = 5.0
+  ymin = 0.0
+  ymax = 1.0
+  use_displaced_mesh = false
+[]
+
+[Variables]
+  [disp_x]
+  []
+  [disp_y]
+  []
+[]
+
+[Physics]
+  [SolidMechanics]
+    [QuasiStatic]
+      displacements = 'disp_x disp_y'
```

Comment:
Why do you need this. I have to do this too?  It makes no sense.  I guess this is why people use global params for displacements.  There is nothing you can do about this but I find it annoying.

Reply:
No idea, but agree super annoying.